tests/Microsoft.Identity.Test.Unit.net45/RequestsTests/IntegratedWindowsAuthUsernamePasswordTests.cs

```diff
@@ -708,6 +755,61 @@ public async Task ManagedUsernameIncorrectPasswordAcquireTokenTestAsync()
                 Assert.AreEqual(0, app.UserTokenCacheInternal.Accessor.GetAllAccessTokens().Count());
             }
         }
+
+        [TestMethod]
+        [TestCategory("IntegratedWindowsAuthAndUsernamePasswordTests")]
+        public async Task UsernamePasswordInvalidClientTestAsync()
```

Comment:
I'm not sure this test is needed, but it's fine now that it's in. Please write another test on MsalServiceExceptionFactoryTest - the object where you actually done the changes. Should be a simple one.